lib/tasks/evm.rake

```diff
@@ -43,8 +43,8 @@ namespace :evm do
 
   task :compile_assets do
     EvmRakeHelper.with_dummy_database_url_configuration do
-      Rake::Task["assets:clobber"].invoke
-      Rake::Task["assets:precompile"].invoke
+      `RAILS_ENV=assets bundle exec rake assets:clobber`
+      `RAILS_ENV=assets bundle exec rake assets:precompile`
```

Comment:
Does this work @skateman 
`RAILS_ENV=assets bundle exec rake assets:clobber assets:precompile`